apps/src/blockly/addons/cdoUtils.ts

```diff
@@ -196,7 +199,7 @@ export function moveHiddenBlocks(
   procedureTypesToHide: string[] = []
 ) {
   if (procedureTypesToHide.length === 0 || !hasBlocks(source)) {
-    return {mainSource: {}, hiddenDefinitionSource: {}};
+    return {mainSource: source, hiddenDefinitionSource: {}};
```

Comment:
Related to the change above, we should just return an unmodified source if we aren't hiding any blocks. This line was previously unreachable.